warehouse/static/sass/blocks/_vertical-tabs.scss

```diff
@@ -183,7 +183,7 @@
     }
 
     @media only screen and (max-width: $tablet) {
-      margin: 25px 10px 10px 10px;
+      margin: 25px 10px 10px;
```

Comment:
kinda gross that the new once prefers this way less obvious three value version just cause r/l are same margin, but OK